pyPostcode/__init__.py

```diff
@@ -39,10 +40,10 @@ def __init__(self, api_key, api_version=(3, 0, 0)):
         if (2, 0, 0) <= api_version < (3, 0, 0):
             self.url = 'https://postcode-api.apiwise.nl'
         else:
-            self.url = 'http://api.postcodeapi.nu'
+            self.url = 'https://api.postcodeapi.nu'
 
     def handleresponseerror(self, status):
-        if status == 401:
+        if status in {401, 403}:
```

Comment:
Dictionary? Why not a list?

Reply:
No particular reason when it's so small, I just have a habit of using sets for lookups because they're fast.
I'll change it to a list to be less confusing.

Reply:
Oh it creates set. Was not aware of this syntax, but that's perfect.